src/app/api/tasks/templates/templates.service.ts

```diff
@@ -50,9 +52,28 @@ export class TemplatesService extends BaseService {
     const policyGate = new PoliciesService(this.user)
     policyGate.authorize(UserAction.Create, Resource.TaskTemplates)
 
+    // If template with same title already exists, append successive "copy" string after it
+    // E.g. "hello" -> "hello copy" -> "hello copy copy"
+    let title = sanitize(payload.title.trim())
+    // For some reason normal parameterized string doesn't work so using Prisma.raw with sanitized strings
+    const query = Prisma.sql`
+      SELECT COUNT(*) AS count
+      FROM "TaskTemplates"
+      WHERE "title" ~ '^${Prisma.raw(`${title}`)}( copy)*$'
```

Comment:
does this protect from sql injection though.. isnt there a way to send as parameters?